### DISCUSSION TITLE
Use in Swift Package?

### DISCUSSION THREAD

**Discussion by jsorge** [dataFude-app/dataFude#12] [Ideas]
Can it be possible to run DataTile with Swift packages? My current itch is Advent of Code and I have a target with the problems for a given day that I'm debugging and would love to use DataTile for the visual.

**Comment by icanzilb** [comment]
@jsorge dataTile for Simulator is only able to stream the logs from the simulator because of Apple's sandbox. However, if you do have an interesting use case (which I think working with packages from the CLI is), DM me on twitter if interested and I could make you a non-sandboxed build to try, I'll also be interested in your experience.

For what is worth — I think using dataTile as an ad-hoc UI for cli tools is one of the most valuable use cases.
